components/Datasets/DescribeDataset.vue

```diff
@@ -355,6 +378,36 @@
               :error-text="getFirstError('tags')"
               :warning-text="getFirstWarning('tags')"
             />
+            <div class="flex items-center gap-4 mt-2 mb-3">
+              <BrandedButton
+                type="button"
+                color="primary"
+                :disabled="isGeneratingTags || !form.title || !form.description || form.description.trim().length === 0"
```

Comment:
Sorry, I didn't catch it on the previous PR, there is a `loading` props to handle the `isGeneratingTags` state. It will disable the button and add a loader